.github/workflows/release-artifacts.yaml

```diff
@@ -25,12 +25,15 @@ jobs:
       uses: olegtarasov/get-tag@v2
       id: tagName
 
-    - name: Build on PR
-      id: docker_build
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.CR_PAT }}
```

Comment:
Looks like this is something we need to create manually this time, I assume you've done this already? Is this a company-shared secret?